Library/ViewModels/LiveStreamDiscoveryViewModel.swift

```diff
@@ -53,8 +59,26 @@ LiveStreamDiscoveryViewModelInputs, LiveStreamDiscoveryViewModelOutputs {
     self.showAlert = projectAndTappedLiveStreamEvent.errors()
       .mapConst(Strings.Couldnt_open_live_stream_Try_again_later())
 
-    let freshLiveStreamEvents = self.isActiveProperty.signal.filter(isTrue)
-      .flatMap { _ in
+    let periodicRefresh = self.viewDidLoadProperty.signal.switchMap {
+      timer(interval: .seconds(60*5), on: AppEnvironment.current.scheduler)
+    }
```

Comment:
i kinda think we don't need this countdown. it would only be useful if the person sat on this page for 5 mins. otherwise navigating away and back should trigger the view will appear.